uberenv.py

```diff
@@ -680,7 +680,7 @@ def find_spack_pkg_path(self, pkg_name, spec = ""):
 
     # Extract the first line of the full spec
     def read_spack_full_spec(self,pkg_name,spec):
-        res, out = sexe("spack/bin/spack spec " + pkg_name + " " + spec, ret_output=True)
+        res, out = sexe("spack/bin/spack spec '" + pkg_name + " " + spec + "'", ret_output=True)
```

Comment:
Would this be easier to read if it used `String.format()` instead of concatenation?

Reply:
Probably, but in hacking mode, I just add the minimum number of symbols required :)
I can change that tomorrow.